design/one-pager-simple-class-selection.md

```diff
@@ -0,0 +1,411 @@
+itself is now provider specific and no longer portable across providers. We
+solved this issue by introducing [portable resource classes].
+
+A portable resource class is effectively an indirection to a non-portable
+resource class. Portable resource classes can be set as the default for a
+particular namespace, and will be used by any resource claims of their
+corresponding kind that do not specify a resource class for dynamic
+provisioning, or a managed resource for static provisioning. This can be thought
+of as "publishing" non-portable resource classes (which may exist in a distinct
+namespace used to logically group infrastructure) to other namespaces, where
+applications may use them to satisfy claims for their infrastructure
+requirements.
+
+This pattern is flexible and powerful, but Crossplane maintainers and [community
+members] have observed that it's verbose; three distinct Kubernetes resources (a
+claim, portable class, and non-portable class) must exist for dynamic
+provisioning to occur. Community members have also [provided feedback] that
+having to create portable resource classes for every new application namespace
+can be onerous.
+
+## Terminology
+
+Most terminology is covered by the [Crossplane glossary]. Key terms used in this
+document include:
+
+* Crossplane uses a _provider_ to satisfy infrastructure needs. Google Cloud
+  Platform (GCP), Elastic Cloud, and Kubernetes are all providers.
+* A resource is said to be _portable_ if it is not tightly coupled to any one
+  provider. Managed resources like `RDSInstance` are coupled to a provider
+  because they expose provider specific configuration parameters, whereas
+  resource claims like `PostgreSQLInstance` are portable because they do not.
+* _Dynamic provisioning_  is the act of satisfying a _portable_ request for
+  infrastructure (i.e. a resource claim) by automatically provisioning a
+  non-portable managed resource.
+* _Resource classes_ enable dynamic provisioning. A resource class determines
+  what class of managed resource should be created to satisfy a resource claim.
+* _Separation of concern_ happens when an opinionated party can publish classes of
+  infrastructure to be consumed by parties unconcerned with such details.
+  Imagine an organisation in which an infrastructure team supports many product
+  teams who own the lifecycle of their infrastructure. Separation of concerns
+  allows the infrastructure team to define a 'production database' class of
+  service that happens to be a high availability, SSD backed GCP CloudSQL
+  instance. Product teams can then self-service by provisioning a production
+  database and trust that it will be configured appropriately.
+* Crossplane recommends using Kubernetes namespaces to logically group
+  infrastructure into an _infrastructure namespace_, for example
+  `production-infra` or `us-east-coast`. This infrastructure could be claimed
```

Comment:
should we remove this bullet since there will only be cluster-scoped infra going forward, or qualify it with Crossplane v0.3 recommends using infrastructure namespaces?

Reply:
Good catch - I actually thought I already deleted this. My thinking was that I'd remove this and rely on the brief mention of infrastructure namespaces at line ~20 and under "Infrastructure Scope" to provide this context.